notebooks/mock-notebooks/login/User Registration.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<ul><li>Yes, we can enforce some secure password standards. We will have to build a password validator and it can raise an error if the password does not match the standards. Do we have something in the UI, if so we can use the same in notebooks, if not we can decide on a common thing (possibly: one upper case, one special character, one numerical value, and minimum 7 characters).</li><li>Either we can make certain fields compulsory during registration itself like Institution. Or prompt the user to update/complete their profile or certain fields in the profile whenever their perform a login. These are two ways I can think of w.r.t notebooks.</li></ul>

Reply:
<p>Great !!! I will add these changes w.r.t Complete profile message with the Account update Flow, cause at that point I also be able to indicate the steps to update information. Will add this reference to the Account update ticket.</p>

Reply:
<p>Got it ^_^</p>